Source/Core/Common/IniFile.cpp

```diff
@@ -133,6 +168,18 @@ bool IniFile::Section::Get(const std::string& key, int* value, int defaultValue)
   return false;
 }
 
+bool IniFile::Section::Get(const std::string& key, s64* value, s64 default_value) const
+{
+  std::string temp;
+  bool retval = Get(key, &temp);
+
+  if (retval && TryParse(temp, value))
```

Comment:
This really isn't that big of a deal (this isn't performant code) but I think this would be a little cleaner:

```
Get(...)
{
*value = default_value;
std::string temp;
bool retvalue = Get(key, &temp);
return retval && TryParse(temp, value);
}
```